### PR TITLE
[NRB-2024] - addition of sponsor and general update

### PR DESCRIPTION
**Pull request by alvo254**
Added strathmore university as platinum sponsor
Update of Nairobi homepage and sponsors page to include sponsor packages
Addition of CFP dates

### CODE REVIEW

File: themes/devopsdays-theme/archetypes/welcome.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ Description = ""
 Title = ""
 Type = "welcome"
 Aliases = [""]
-+++
++++
```
Comment (adrianmoisey) [inline themes/devopsdays-theme/archetypes/welcome.md:6]:
```suggestion
+++
```

There's a missing trailing newline here.
It's a nitpick, but it shouldn't be in this PR

File: content/events/2024-nairobi/welcome.md [hunk 2]
```diff
@@ -81,6 +96,15 @@ Description = "devopsdays Nairobi 2024"
   </div>
 </div>
 
+
+<div class = "row">
+  <div class = "col-md-2">
+    <strong>Loction</strong>
```
Comment (adrianmoisey) [inline content/events/2024-nairobi/welcome.md:102]:
```suggestion
    <strong>Location</strong>
```

Typo

File: data/events/2024/nairobi/main.yml [hunk 3]
```diff
@@ -14,11 +14,11 @@ startdate:  2024-06-01T08:00:00+03:00 # The start date of your event. Leave blan
 enddate:  2024-06-01T18:00:00+03:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
 
 # Leave CFP dates blank if you don't know yet, or set all three at once.
-cfp_date_start:  # start accepting talk proposals.
-cfp_date_end:  # close your call for proposals.
+cfp_date_start:  2024-04-01T08:00:00+03:00 # start accepting talk proposals.
+cfp_date_end:  2024-04-01T18:00:00+03:00 # close your call for proposals.
```
Comment (adrianmoisey) [inline data/events/2024/nairobi/main.yml:18]: The CFP start and end date are the same here. Is that an error?

Comment (alvo254) [reply data/events/2024/nairobi/main.yml:18]: Yes thanks let me fix that.

**Pull request merged**